content/blog/lektor-32-released/contents.lr

```diff
@@ -0,0 +1,59 @@
+
+!! Since Python 2 is EOL and its [final release](https://blog.python.org/2020/04/python-2718-last-release-of-python-2.html) ever happened on April 20th, 2020 this will be the last Lektor release with Python 2 support.
+
+A big shoutout to all the contributors who helps us make Lektor awesome!
+
+## Bugfixes
+
+* Bugfix to correctly calculate relative urls from slugs that contain dots.
+* Bugfix to allow negative integers in integer fields in the admin UI.
+* Bugfix where `lektor plugins reinstall` triggered `on_setup_env` instead of just reinstalling plugins.
+
+
+## New features
+
+* Added the ability to [generate video thumbnails](/docs/templates/videoops/) with ffmpeg.
+
+
+## Improvements
+
+* Added support for deleting and excluding files for the rsync deployment publisher. (thank you [m-lib](https://github.com/m-lib))
+* Several modernization and performance improvements to the admin UI (thank you [Jakob Schnitzer](https://github.com/yagebu))
+* Improved speed of source info updates. (thank you [Jeff Dairiki](https://github.com/dairiki))
+* Set colorspace to sRGB for thumbnails. This should result in smaller thumbnails for some cases where the source image uses a more exotic colorspace.
+* Now stripping profiles and comments from thumbnails.
+* Improved speed of flow rendering in the admin UI. (thank you [David Ferguson](https://github.com/davidferguson))
+* Improved image-heavy build speeds by reducing the amount of data extracted from EXIFs.
+* Added the ability to collapse flow elements in the admin UI. (thank you [jtraub91](https://github.com/jtraub91))
+* Now `extra_flags` is passed to all plugin events.
+* Extra flags can now be passed to the `clean` and `dev shell` CLI commands.
+* Added `mode` and `upscale` thumbnail arguments, changing the preferred method to crop to using `mode`. `mode` can be `crop`, `fit`, or `stretch`. `upscale=False` can now prevent upscaling.
+* Added a new CLI command `lektor dev new-theme`.
+* Made admin use full UTF-8 version of RobotoSlab. Fixes missing glyphs for some languages (thank you [Jakob Schnitzer](https://github.com/yagebu))
+* Bumped minimum Jinja2 version to 2.11
+* Bumped filetype dependency to 1.0.7 because of API changes
+* Relative urls are now as short as possible.
+* Automatically include setup.cfg configured for universal wheels when creating plugins
+* Changed default slug creation to use [slugify](https://github.com/un33k/python-slugify). This should mean greater language support, but this may produce slightly different results than before for some users
+
+!!!! The slugify change may mean that your URLs will change when you update to Lektor 3.2. This could lead to broken links from external sites such as search engines. This can be fixed by providing a custom slug for the necessary pages. We don't have any plans to change the slug algorithm in the future.
```

Comment:
Maybe add a

`IMPORTANT`  header?

Reply:
Not sure if I can do that inside of those admonition boxes, see render. I wanted something that pops out but maybe its not great to have it at the bottom?

Reply:
Ahhhh right! admonition box is fine, I missed the `!!!!`